### ISSUE TITLE
Shadowlands?

### ISSUE THREAD

**Issue by MuTLY** [Goranaws/Dominos_Units#1]
Do you plan on fixing the addon for Shadowlands? Right now, I'm using Z-Perl to move units around, but your addon seems to be much simpler to use, it would be great to have it working or maybe integrated directly into Dominos. 

If the following units could be on a working condition and movable by default, I bet yours would be one of the most used addons all-around:

- Player
- Player target
- Target of target
- Player pet
- Player pet target
- Player focus
- Arena enemy frames
- Boss frames

On a side note, I'm currently using your Dominos_Auras addon and it's super useful.

**Comment by Goranaws** [comment]
I do plan on releasing an update for shadowlands... I've been focusing on my other add-ons first, as units will be the most involved to get up and running. I have a beta version available, if you want to test it out.

**Comment by MuTLY** [comment]
Yes that would be awesome, @Goranaws - just point me to it. Thank you!

**Comment by Goranaws** [comment]
I've just uploaded my personal working version on here.  You can download from here now.

**Comment by Goranaws** [comment]
Do note that it needs some heavy polish right now. The options menu has issues, but should still work without errors.

**Comment by Goranaws** [comment]
I've posted an update that should fix all existing issues. If you encounter any issues, please let me know. I'll also be updating curse with this update.

**Comment by MuTLY** [comment]
This is great news. I'll update asap.

**Comment by Goranaws** [comment]
If you have had a chance to test the new version, have you encountered any issues? I am trying to make Domino's units as stable as possible, before I begin working on its replacement. The more stable it is, the easier it will be to translate over

**Comment by MuTLY** [comment]
> If you have had a chance to test the new version, have you encountered any issues? I am trying to make Domino's units as stable as possible, before I begin working on its replacement. The more stable it is, the easier it will be to translate over

Hi sorry for the late reply. I've been using for a about a week now, and haven't seen any issues with it apart from, sometimes, the health bar gets transparent instead of having a solid color. I'm running WoW on a 2nd monitor, by the way. I run it on a laptop,  maybe the scaling thing on Windows has something to do with it?

**Comment by MuTLY** [comment]
Actually, it's not the health bar. It's the character name label, like this:

![image](https://user-images.githubusercontent.com/777006/111399455-2c3ebe00-86a4-11eb-9d69-cd50194ffe98.png)

I think it should have a solid background color, or some texture.

**Comment by Goranaws** [comment]
This could be a bug. I will need to revisit how I have coded the background. It is supposed to change color based on your what you are targeting. Green for friend, red for enemy. I have the feature disabled in my UI, so I wouldn't have seen it glitching. 

And just to be sure, go into the frames settings, and make sure that the background for the entire frame is enabled.(not the names background)

**Comment by MuTLY** [comment]
Background of the frame was indeed disabled (unchecked) by default. Once checked, it's now showing a transparent bar behind the character name. Thanks!

![image](https://user-images.githubusercontent.com/777006/111850059-f68d1580-88ed-11eb-827a-00a380b08123.png)
